src/css/button.scss

```diff
@@ -5,3 +5,42 @@ a.button {
 a.button-primary {
   @extend .btn-primary;
 }
+
+.btn {
```

Comment:
@maymax777 , this file is duplicated with PR [#356](https://github.com/keep-network/website/pull/356/files) .